### DISCUSSION TITLE
Is local disk have same effect like object storage or it just mock test for quick start?

### DISCUSSION THREAD

**Discussion by dayiguizhen** [juicedata/juicefs#279] [Q&A] [answered]
I'm try to use local disk format a volume,and mount this volume on two machine.
I write one file in one machine,and try to read in another machine.
But when I use `vim` open that file, operating system is block, I even can not use ssh to connect.So I have to reboot this machine by push power button.
I'm use wrong command? Or disk local mount can't share for anther machine?

**Comment by xiaogaozi** [comment, marked as answer]
The local disk just for testing purpose, you need use object storage to sharing files.

**Reply by dayiguizhen** [reply]
Thx.If I want do benchmark, multiprocess parallelism run `dd` is a good idea?

**Reply by dayiguizhen** [reply]
I use minio for object storage.

**Reply by xiaogaozi** [reply]
Running `dd` is ok.
